### DISCUSSION TITLE
No PCAPs in the PCAP section

### DISCUSSION THREAD

**Discussion by Tredmond7511** [Security-Onion-Solutions/securityonion#8545] [Unsupported Versions]
I'm new to security onion 2 and just installed a distributed deployment with a manager, forward sensor, and search node. I'm getting traffic on the span port of the sensor and I see pcaps in the nsm folder. However, there's nothing in the pcap folder on the site and no pcaps in the kibana events I do see.

**Comment by isaacgolding** [comment]
Ok the pcap link on the site is going to be blank by default.   You have to hit the big blue + icon and define a job in order to pull info from the NSM datastore.   Basically the PCAP page is there to allow you to extract/collect data in PCAP form that meets your criteria.   

The ES Kibana interface is there to allow you filter and search the parsed database info that has been collected.   And if I'm not mistaken its not going to give you raw pcap output.    You would need to go back to the PCAP link OR to a dedicated analyst workstation to sift through raw PCAP data and analyze it manually.   

Where you will probably want to really start working on SO would be the alerts & playbook sections.     That is where all the "easy" action can be found.  There is a lot of information in those two areas alone to keep most people busy for weeks/months.      I'd also suggest going through the introduction docs (again?).   If you are really new then SO can be overwhelming in its complexity.  Ive gone back a few times and re-read a lot of the documentation to try to pick up stuff that I missed the first few times.

[https://docs.securityonion.net/en/2.3/introduction.html](https://docs.securityonion.net/en/2.3/introduction.html)

**Reply by Tredmond7511** [reply]
Thank you for the reply. Your advice worked but raised more questions. So to get this straight that security onion 2 doesn't upload pcaps automatically to kibana like the old security onion used to? I have to create a job every time I want pcaps now? Is there a way so it occurs automatically? Was there a purpose to this change? The old security onion worked much smoother in this regard.

**Comment by isaacgolding** [comment]
The data is fed into Kibana.   The PCAP page is there so you can manually extract PCAP data.   If you take a look at https://docs.securityonion.net/en/2.3/soc.html I think you will find it does a good job of explaining each piece of the puzzle.

**Reply by Tredmond7511** [reply]
So my issue is that I don't think my PCAPS are getting fed into kibana. I go to the network events at the bottom and the only mention I see of PCAPS is the far right column that says pivot to hunt or PCAPS. I click the link and it takes me to hunt but there is no option from there to pivot to pcaps. I've read the documentation but it gives me no insight

**Comment by dougburks** [comment]
If you're starting from Kibana, here's the workflow:
- find an event of interest and click the `Hunt and optionally pivot to PCAP/Cases` link
- the Hunt interface appears and displays that same event
- click one of the fields in the event (example: `source.ip`) to get the action menu
- on the action menu, click the `PCAP` action

For more information, please see:
https://docs.securityonion.net/en/2.3/kibana.html#pcap-cases
https://docs.securityonion.net/en/2.3/dashboards.html#actions
https://docs.securityonion.net/en/2.3/pcap.html
